packages/samples/headless-react/src/pages/SearchPage.tsx

```diff
@@ -0,0 +1,373 @@
+import './SearchPage.css';
```

Comment:
I'll highlight the changes in this file. Notably, I removed most of the create-react-app boilerplate such as the logo.